### ISSUE TITLE
Switch - Ensure parts of graphical objects essential for understanding content have sufficient contrast - (2036570198)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#7823]
### Violation: 
 Ensure parts of graphical objects essential for understanding content have sufficient contrast 

### WCAG Reference: 
- [1.4.11 Non-test Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) 

### Severity: 
 5 

### Media Type: 
 Color and Contrast 

--- 
### Areas for Remediation: 
- Title: Switch - Ensure parts of graphical objects essential for understanding content have sufficient contrast - (2036570198) 
- Module: Switch 
**Issue**
There are graphical objects with a color contrast ratio below 3.00:1, when checking with "Toggle theme" on.

Foreground: #757575
Background: #515151
Contrast ratio: 1.7:1

Examples include:
- Switch mages. 

**User Impact**
Users with low vision will have difficulty identifying this content.

 
**Suggestion**
Ensure parts of graphical objects essential for understanding content have sufficient contrast. The required minimum contrast ratio is 3.00:1.

Common examples of qualifying objects include lines in a chart, meaningful icons, and annotations within an image.

Graphics that require particular presentation to preserve their meaning are exempt from this requirement.

To evaluate color contrast, see the Accessible Color Picker extension: https://www.accessibility.dev/

 
--- 
### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Duplicate of #7822
